sigstore/_internal/rekor/client.py

```diff
@@ -71,10 +106,47 @@ def from_response(cls, dict_: Dict[str, Any]) -> RekorEntry:
             integrated_time=entry["integratedTime"],
             log_id=entry["logID"],
             log_index=entry["logIndex"],
-            verification=entry["verification"],
-            raw_data=entry,
+            inclusion_proof=RekorInclusionProof.parse_obj(
+                entry["verification"]["inclusionProof"]
+            ),
+            signed_entry_timestamp=entry["verification"]["signedEntryTimestamp"],
         )
 
+    @classmethod
+    def from_bundle(cls, dict_: Dict[str, Any]) -> RekorEntry:
+        """
+        Creates a `RekorEntry` from an offline Rekor bundle.
+
+        See: <https://github.com/sigstore/cosign/blob/main/specs/SIGNATURE_SPEC.md#properties>
+        """
+
+        payload = dict_["payload"]
+        return cls(
+            uuid=None,
+            body=payload["body"],
+            integrated_time=payload["body"],
```

Comment:
How come everything is being initialised with `payload["body"]`?

Reply:
(This did make me realize that the `payload` key wasn't correctly capitalized, though, so thanks for that!)

Reply:
What I mean is, shouldn't it read something like:
```python
integrated_time=payload["integratedTime"],
```

Reply:
Oh yep, that's another copy-paste error 😅 

Thanks for catching, and sorry for the initial misunderstanding!